bin/install-deps.sh

```diff
@@ -22,9 +22,17 @@ YQ_VERSION=3.3.0
 sudo yum -y install net-tools bind-utils wget unzip git
 
 #
-# download the yq container dependency
+# checkout and build the yq container dependency
 #
-wget -O $CCPROOT/yq https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64
+mkdir /tmp/yq
+cd /tmp/yq
+git clone https://github.com/mikefarah/yq.git
+cd yq
+git checkout ${YQ_VERSION}
+go build
```

Comment:
I think a lot of this can be replaced with `go install`.

https://github.com/mikefarah/yq/releases/tag/3.3.0 = `3f913afbb9ed035bbd1df21c7c89c5525784881f`

`go version -m` shows `v3.0.0-20200417222338-3f913afbb9ed` below:

```console
$ mkdir tmp-dst
$ GOBIN="$(pwd)/tmp-dst" go install github.com/mikefarah/yq/v3@3.3.0
$ go version -m tmp-dst/yq
tmp-dst/yq: go1.17.5
        path    github.com/mikefarah/yq/v3
        mod     github.com/mikefarah/yq/v3      v3.0.0-20200417222338-3f913afbb9ed      h1:OPcaXkq4IrtwnP0leTxMFv9f6R9HVNmK9ryr80sWUEM=
        dep     github.com/fatih/color  v1.9.0  h1:8xPHl4/q1VyqGIPif1F+1V3Y3lSmrq01EabUW3CoW5s=
        dep     github.com/goccy/go-yaml        v1.4.3  h1:+1jK1ost1TBEfWjciIMU8rciBq0poxurgS7XvLgQInM=
        dep     github.com/kylelemons/godebug   v1.1.0  h1:RPNrshWIDI6G2gRW9EHilWtl7Z6Sb1BR0xunSBf0SNc=
        dep     github.com/mattn/go-colorable   v0.1.6  h1:6Su7aK7lXmJ/U79bYtBjLNaha4Fs1Rg9plHpcH+vvnE=
        dep     github.com/mattn/go-isatty      v0.0.12 h1:wuysRhFDzyxgEmMf5xjvJ2M9dZoWAXNNr5LSBS7uHXY=
        dep     github.com/pkg/errors   v0.9.1  h1:FEBLx1zS214owpjy7qsBeixbURkuhQAwrK5UwLGTwt4=
        dep     github.com/spf13/cobra  v1.0.0  h1:6m/oheQuQ13N9ks4hubMG6BnvwOeaJrqSPLahSnczz8=
        dep     github.com/spf13/pflag  v1.0.5  h1:iy+VFUOCP1a+8yFto/drg2CJ5u0yRoB7fZw3DKv/JXA=
        dep     golang.org/x/sys        v0.0.0-20200413165638-669c56c373c4      h1:opSr2sbRXk5X5/givKrrKj9HXxFpW2sdCiP8MJSKLQY=
        dep     golang.org/x/xerrors    v0.0.0-20191204190536-9bdfabe68543      h1:E7g+9GITq07hpfrRu66IVDexMakfv52eLZ2CXBWiKr4=
        dep     gopkg.in/op/go-logging.v1       v1.0.0-20160211212156-b2cb9fa56473      h1:6D+BvnJ/j6e222UW8s2qTSe3wGBtvo0MbVQG/c5k8RE=
        dep     gopkg.in/yaml.v3        v3.0.0-20200313102051-9f266ea9e77c      h1:dUUwHk2QECo/6vqA44rthZ8ie2QXMNeKRTHCNY2nXvo=
```